ops/production-deploy.tmpl.yaml

```diff
@@ -129,8 +129,12 @@ extraEnvVars: &envVars
     value: b2.adventistdigitallibrary.org
   - name: HYKU_ADMIN_ONLY_TENANT_CREATION
     value: "true"
+  - name: HYKU_ATTACK_RATE_THROTTLE_OFF
+    value: "true"
   - name: HYKU_ALLOW_SIGNUP
     value: "false"
+  - name: HYKU_ATTACK_RATE_THROTTLE_OFF
```

Comment:
looks like you have two of these?

Reply:
oops! I copied it in and then tried to move it to alphabetize it...